packages/scaffolding-cli/templates/build/azDevops/azure/templates/steps/build-netcore.yml

```diff
@@ -19,13 +19,7 @@ parameters:
   vulnerability_workdir: ""
   #  Static Code Analysis
   static_code_analysis: false
-  static_code_analysis_build_path: ""
-  Sonar_serviceConnection: ""
-  Sonar_organization: ""
-  Sonar_scannerMode: ""
-  Sonar_projectKey: ""
-  Sonar_projectName: ""
-  Sonar_projectVersion: ""
+  sonar_container: ""
```

Comment:
I think we are not using this one yet, but 👍 - though not sure we can use the scanner.